build_for_portal.py

```diff
@@ -455,32 +455,39 @@ def reformat_for_drupal(info):
 
         ensure_directory(images_dir)
 
+        # ADDED 21 Jan 2025: selective processing of images
+        # the set of file names is to be stored in imagefiles
+        # The initial value includes images defined in attributes (to copy every time)
+        imagefiles = {"kebab.png","app-launcher.png","red-hat-applications-menu-icon.jpg","delete.png"}
```

Comment:
Why are we hard coding these values, and is there a risk that we miss new additions going forward? Also, are these uniform across all branches? Maybe we need to scan the attributes folder for images.

Reply:
I have reviewed the logic more carefully and realized that the prepopulation is unnecessary as the attributes file is scanned as part of looping through the includes. Regression testing showed that if I removed the prepopulation, the only difference was the disappearance of these four images from the API books, which otherwise have no images at all; a build-test of these books using asciidoctor-pdf shows that no issues are caused by removing these images.